### PR TITLE
fix(combo-box): overwrite icon color on disabled state in solar theme

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description
This fix also solved the issue of the arrow icon on tree-select that did not display the icon same as combo-box. The color of the arrow icon in the disabled state is similar to the background. So, I overwrote the style of the disabled state and specific on the charcoal. 

Fixes # (issue)
ELF-1674 'Arrow' icon is not displayed in the disabled state on Solar theme(charcoal)
 
## Type of change
- [x] Bug fix (non-breaking change which fixes an issue)

## Checklist
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

**Review by dtanp-rft — CHANGES_REQUESTED** [review]
Style should be consistent with button disabled state.
```
  &[disabled] [part~=icon] {
    opacity: 0.4;
    color: @button-disabled-text-color;
  }
```

**Comment by bualoy-napat** [comment]
@dtanp-rft I have already updated as your request.

**Review by dtanp-rft — CHANGES_REQUESTED** [review]
Can we add one unit test to test about `disabled`? Basically, set disabled on combobox and it should set disabled to text-field.

**Comment by bualoy-napat** [comment]
@dtanp-rft I have already added the unit test and renamed the misspelling file name.

**Review by dtanp-rft — APPROVED** [review]

File: packages/elements/src/combo-box/__test__/combo-box.template.test.js [hunk 1]
```diff
@@ -62,4 +62,23 @@ describe('combo-box/Template', () => {
...
+    it('Input should be disabled when disabled attribute is set', async () => {
+      const el = await fixture('<ef-combo-box disabled></ef-combo-box>');
+      const input = el.shadowRoot.querySelector('[part=input]');
+
+      await elementUpdated(el);
+
+      expect(input.disabled).to.be.equal(true)
+    });
+
+    it('Input should be disabled when readonly attribute is set', async () => {
```
[the remark this answers is not in the document]
Comment (bualoy-napat) [reply packages/elements/src/combo-box/__test__/combo-box.template.test.js]: I have fixed it with the current commit.

**Pull request merged**